application/.env

```diff
@@ -37,3 +37,9 @@ MAPBENDER_DATABASE_URL="sqlite:///%kernel.project_dir%/var/db/demo.sqlite"
 ###> mapbender/mapbender ###
 KERNEL_CLASS="\Mapbender\BaseKernel"
 ###< mapbender/mapbender ###
+
+###> lexik/jwt-authentication-bundle ###
+JWT_SECRET_KEY=%kernel.project_dir%/config/jwt/private.pem
+JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
+JWT_PASSPHRASE=b62cfe7b2ee8582654e0d2a6a677a0014806194640a6b93ff359cbd5452c2cf2
```

Comment:
I don't think this should be committed. I suggest to have a placeholder here (something like "change me") and than have an additional command in the bootstrap script that auto-generates a passphrase and places it in the .env.local file if it does not yet exist